test/src/node.rs

```diff
@@ -89,6 +90,14 @@ impl Node {
         modifier(&mut app_config);
         fs::write(&app_config_path, toml::to_string(&app_config).unwrap()).unwrap();
 
+        // write sentry section into ckb configuration file
+        // so that we can run integration test for ckb which builds with or without sentry feature
+        let mut file = fs::OpenOptions::new()
+            .append(true)
+            .open(&app_config_path)
+            .unwrap();
+        writeln!(file, "\n[sentry]\ndsn = \"\"").unwrap();
+
```

Comment:
https://github.com/nervosnetwork/ckb/blob/develop/test/template/ckb.toml#L18-L23 the test-used `ckb.toml` has configured `[sentry]`, why attach a new one here?